service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -213,29 +246,27 @@ private CallbackRequest buildCallbackRequest(OrderStatus status) {
 
     @Nested
     class StateChangeTests {
-        private final String event = "internal-changeState:Gatekeeping->PREPARE_FOR_HEARING";
```

Comment:
Why move this outside of the nested class, this is the only place it is used

Reply:
Used it at some point but forgot to remove, good catch, fixed. :+1: